HW_Multithreading/src/Main.java

```diff
@@ -1,2 +1,23 @@
 public class Main {
+    public static void main(String[] args) throws InterruptedException {
+
+        Counter count=new Counter(10);
+
+        Runnable myThread1=new MyThread(count);
+        Runnable myThread2=new MyThread(count);
+
+        Thread t1=new Thread(myThread1);
+        Thread t2=new Thread(myThread2);
+
+        t1.start();
+        t1.join();
```

Comment:
this is wrong: main thread is waiting for the t1 to finish and only then you start t2. We need 2 threads started almost simultaneously. So, you do not have 2 threads competing for the same Counter instance and you will always get correct result (0) because threads run in a sequence, not in parallel.